ocs_ci/deployment/disconnected.py

```diff
@@ -423,16 +499,19 @@ def prepare_disconnected_ocs_deployment(upgrade=False):
         # try to use it also there.
         # https://cloud.redhat.com/blog/how-oc-mirror-will-help-you-reduce-container-management-complexity
 
-        icsp_file = get_and_apply_icsp_from_catalog(image=index_image, apply=False)
-        icsp = {}
-        if icsp_file:
-            with open(icsp_file) as f:
-                icsp = yaml.safe_load(f)
+        # icsp_file = get_and_apply_icsp_from_catalog(image=index_image, apply=False)
+        idms_file = get_and_apply_idms_from_catalog(image=index_image, apply=False)
+        # icsp = {}
+        idms = {}
+        if idms_file:  # icsp_file:
+            with open(idms_file) as f:
+                idms = yaml.safe_load(f)
 
         mirrored_index_image = mirror_index_image_via_oc_mirror(
             index_image,
             constants.DISCON_CL_REQUIRED_PACKAGES_PER_ODF_VERSION[get_ocp_version()],
-            icsp,
+            idms,
+            # icsp,
```

Comment:
You have to pass the `idms` as named argument:
```suggestion
            idms=idms,
```